src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -759,6 +900,23 @@ bool isLambdaConversion(BoundExpression expr)
                 => expr is BoundConversion conv && conv.ConversionKind == ConversionKind.AnonymousFunction;
         }
 
+        private static RefKind GetPatchedRefKind(RefKind argRefKind, RefKind paramRefKind)
```

Comment:
nit: this is only used once (above), could just be a local function?

Reply:
I'll just re-inline the code, I was using it in multiple places before.